doc/config.yml

```diff
@@ -59,26 +89,51 @@ Extensions:
         executable: ${ROOT_DIR}
         remove:
             framework: !include ${MOOSE_DIR}/framework/doc/remove.yml
+            cardinal:
+                - /ICs/BulkEnergyConservationIC
+                - /Cardinal/ICs/BulkEnergyConservation/BulkEnergyConservationICAction
+                - /Cardinal/ICs/VolumetricHeatSource/VolumetricHeatSourceICAction
     MooseDocs.extensions.common:
         shortcuts: !include ${MOOSE_DIR}/framework/doc/globals.yml
     MooseDocs.extensions.acronym:
         acronyms:
             framework: !include ${MOOSE_DIR}/framework/doc/acronyms.yml
+            cardinal: !include ${ROOT_DIR}/cardinal/doc/acronyms.yml
             fenix:
                 FENIX: Fusion ENergy Integrated multiphys-X
                 PIC: particle-in-cell
                 CFD: computational fluid dynamics
                 CAD: computer-aided design
     MooseDocs.extensions.sqa:
         active: true
-        categories:
+        categories: # commented-out categories are not available
             fenix: !include ${ROOT_DIR}/doc/sqa_fenix.yml
             framework: !include ${MOOSE_DIR}/framework/doc/sqa_framework.yml
-            ray_tracing: !include ${MOOSE_DIR}/modules/ray_tracing/doc/sqa_ray_tracing.yml
+            #chemical_reactions: !include ${MOOSE_DIR}/modules/chemical_reactions/doc/sqa_chemical_reactions.yml
             electromagnetics: !include ${MOOSE_DIR}/modules/electromagnetics/doc/sqa_electromagnetics.yml
+            fluid_properties: !include ${MOOSE_DIR}/modules/fluid_properties/doc/sqa_fluid_properties.yml
+            heat_transfer: !include ${MOOSE_DIR}/modules/heat_transfer/doc/sqa_heat_transfer.yml
+            misc: !include ${MOOSE_DIR}/modules/misc/doc/sqa_misc.yml
+            navier_stokes: !include ${MOOSE_DIR}/modules/navier_stokes/doc/sqa_navier_stokes.yml
+            phase_field: !include ${MOOSE_DIR}/modules/phase_field/doc/sqa_phase_field.yml
+            reactor: !include ${MOOSE_DIR}/modules/reactor/doc/sqa_reactor.yml
+            rdg: !include ${MOOSE_DIR}/modules/rdg/doc/sqa_rdg.yml
+            ray_tracing: !include ${MOOSE_DIR}/modules/ray_tracing/doc/sqa_ray_tracing.yml
+            scalar_transport: !include ${MOOSE_DIR}/modules/scalar_transport/doc/sqa_scalar_transport.yml
+            solid_mechanics: !include ${MOOSE_DIR}/modules/solid_mechanics/doc/sqa_solid_mechanics.yml
+            solid_properties: !include ${MOOSE_DIR}/modules/solid_properties/doc/sqa_solid_properties.yml
             stochastic_tools: !include ${MOOSE_DIR}/modules/stochastic_tools/doc/sqa_stochastic_tools.yml
+            thermal_hydraulics: !include ${MOOSE_DIR}/modules/thermal_hydraulics/doc/sqa_thermal_hydraulics.yml
+            tmap8: !include ${ROOT_DIR}/tmap8/doc/sqa_tmap8.yml
+            #cardinal: !include ${ROOT_DIR}/cardinal/doc/sqa_cardinal.yml
```

Comment:
Why is Cardinal commented out?

Reply:
Cardinal doesn't use the MOOSE SQA system.

Reply:
I also say this in a comment on Line 109